CHANGELOG.md

```diff
@@ -2,10 +2,11 @@
 
 * In general follow (https://docs.npmjs.com/getting-started/semantic-versioning) versioning.
 
-## <next patch>
+## 2.0.1

```

Comment:
According to semantic versioning this would be 2.1.0 since MultiSelect is new functionality: "MINOR version when you add functionality in a backwards-compatible manner"